app/src/main/java/com/example/tasky/agenda/domain/GetInitialsUseCase.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.tasky.agenda.domain
+
+import javax.inject.Inject
+
+class GetInitialsUseCase @Inject constructor() {
```

Comment:
If a class doesn't need constructor arguments and doesn't have internal state, you can make it an object 👍

Reply:
But then I cannot inject it to the viewModel, I can access it as `GetInitialsUseCase(name)`. In this particular case the logic is all kotlin code, but what if I use Android dependenceis, the I wont be able to make a fake out of it

Reply:
Why would you need to inject an object somewhere? You can just call it the same way and also override the invoke operator.
Android dependencies also don't belong in use cases by definition (since they're part of domain which is isolated). And objects can also implement interfaces btw to serve as implementation for an abstraction (which doesn't need to be done here)